pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -910,7 +910,28 @@ static pj_status_t get_rem_addrs(dtls_srtp *ds,
     pjmedia_sdp_attr *a;
     int af = pj_AF_UNSPEC();
 
-    PJ_UNUSED_ARG(ds);
+    /* Init RTP & RTCP address */
+    pj_bzero(rem_rtp, sizeof(*rem_rtp));
+    pj_bzero(rem_rtcp, sizeof(*rem_rtcp));
+
+    /* If underlying transport is ICE, get remote addresses from ICE */
+    if (ds->use_ice) {
+	pjmedia_transport_info info;
+	pjmedia_ice_transport_info *ice_info;
+
+	pjmedia_transport_info_init(&info);
+	pjmedia_transport_get_info(ds->srtp->member_tp, &info);
+	ice_info = (pjmedia_ice_transport_info*)
+		   pjmedia_transport_info_get_spc_info(
+				    &info, PJMEDIA_TRANSPORT_TYPE_ICE);
+	if (ice_info) {
+	    *rem_rtp = ice_info->comp[0].rcand_addr;
+	    if (ice_info->comp_cnt > 1)
+		*rem_rtcp = ice_info->comp[1].rcand_addr;
+
+	    goto check_rtcp_mux;
```

Comment:
so far we use goto for purposes such as `on_return` or `on_error`, so perhaps it's better to avoid this. especially since here it seems we can just use `else` instead.

Reply:
The goto is for avoiding 'noise' in the diff (no change other than indentation). When examining change history, sometime we need to compare changes line by line and as there will be ~45 lines to be shifted right, I think it is annoying to compare all 45 lines to eventually see nothing changed there. Should it be changed to using `else`?

Reply:
If only the indentation changes, I believe the CR will only mark it with `>>>`, or is this only valid for the old CR (I can't remember)? But anyway, since the codebase clarity and cleanliness is more important than the cleanliness of CR diff, I vote for `else`. Otherwise, we will potentially see a lot of `goto` in the future in order to minimise patch changes.

Reply:
Btw, I just tried it: the option "Hide white space changes" (available in the "Files changed" tab) does the trick to hide indentation modifs.